rita_client/src/dashboard/neighbors.rs

```diff
@@ -106,42 +110,42 @@ fn generate_neighbors_list(
         }
         let neigh_route = maybe_route.unwrap();
 
-        if let Some(stats_entry) = stats.get(&neigh_route.iface) {
-            if get_current_selected_exit().is_some() {
-                let exit_ip = get_current_selected_exit().unwrap();
-                let maybe_exit_route =
-                    get_route_via_neigh(identity.mesh_ip, exit_ip, &route_table_sample);
-
-                // We have a peer that is an exit, so we can't find a route
-                // from them to our selected exit. Other errors can also get
-                // caught here
-                if maybe_exit_route.is_err() {
-                    output.push(nonviable_node_info(
-                        nickname,
-                        neigh_route.metric,
-                        identity.mesh_ip.to_string(),
-                        *identity,
-                        neigh.speed_limit,
-                    ));
-                    continue;
-                }
-                // we check that this is safe above
-                let exit_route = maybe_exit_route.unwrap();
+        let exit_ip = current_exit.map(|e| get_default_exit_ip(e.clone()));
```

Comment:
why map versus just passing the current exit to get default exit ip?

Reply:
Clippy, if if exitServer is none, return None